budgetkey_data_pipelines/tests/common.py

```diff
@@ -0,0 +1,50 @@
+from unittest import TestCase
+from datapackage_pipelines.utilities.lib_test_helpers import ProcessorFixtureTestsBase
+import os
+import json
+import logging
+
+
+class BudgetkeyProcessorsFixturesTests(ProcessorFixtureTestsBase):
+
+    def __init__(self, fixtures_path, fixtures_options):
+        super(BudgetkeyProcessorsFixturesTests, self).__init__(fixtures_path)
+        self._fixtures_options = fixtures_options
+
+    def _get_procesor_env(self):
+        env = super(BudgetkeyProcessorsFixturesTests, self)._get_procesor_env()
+        env["PYTHONPATH"] = os.path.join(os.path.dirname(__file__), os.pardir, os.pardir)
+        return env
+
+    def _get_processor_file(self, processor):
+        processor = processor.replace('.', '/')
+        return os.path.join(os.path.dirname(__file__), os.pardir, processor.strip() + '.py')
+
+    def _replace_data_input_tokens(self, data_in):
+        """preliminary support for replacing tokens inside the fixture data input"""
+        for file in self._fixtures_options.get("file_input_tokens", []):
+            if bytes('{{"__LOAD_FROM_FILE__": "{}"}}'.format(file).encode("utf-8")) in data_in:
+                with open(os.path.join(os.path.dirname(__file__), file)) as f:
+                    data = b'\n'.join([bytes(json.dumps({"data": line}).encode("utf-8")) for line in f.readlines()])
```

Comment:
This is quite hacky - either we store the original file as that (with the `data` column) or we change the name so that it better represents the transformation you're making.